### PR TITLE
Add more detail to Support for `sections`.

### PR DESCRIPTION
**Pull request by jonathanpberger**

### CODE REVIEW

**Comment by bronson** [comment]
There’s an extra quote at the end of your patch.

File: docs/user/features/wikilinks.md [hunk 1]
```diff
@@ -20,7 +20,11 @@ Remember, with `CTRL/CMD+click` on a wikilink you can navigate to the note, or c
 
 ## Support for sections
 
-Foam supports autocompletion, navigation, embedding and diagnostics for note sections. Just use the standard wiki syntax of `[[resource#Section Title]]`.
+Foam supports autocompletion, navigation, embedding and diagnostics for note sections. Just use the standard wiki syntax of `[[resource#Section Title]]`. 
+- If it's an external file, `[your link will need the filename](other-file.md#that-section-I-want-to-link-to)`, but
+- if it's an anchor within the same document, `[you just need an octothorpe and the section name](#that-section-above)`.  
+- Doesn't matter what heading-level the anchor is; whether you're linking to an `H1` like `# MEN WALK ON MOON` or an `H2` like `## Astronauts Land on Plain`, the link syntax uses a single octothorpe: `[Walk!](#men-walk-on-moon)` and `[Land!](#astronauts-land-on-plain-collect-rocks-plant-flag)`. Autocomplete is your friend here.
+`
```
Comment (riccardoferretti) [inline docs/user/features/wikilinks.md:27]:
```suggestion
```

**Review by riccardoferretti — APPROVED** [review]
Thanks for this, I have fixed the extra quote and will merge the doc

**Comment by riccardoferretti** [comment]
@allcontributors add @jonathanpberger for docs

**Comment by riccardoferretti** [comment]
Good ;)

**Pull request merged**